backend/question.py

```diff
@@ -6,3 +6,73 @@
 @question_page.route('/question/ping', methods=['GET'])
 def question_ping():
     return "Pong", 200
+
+# allow user to post a question and word account not more than 200 words
+@question_page.route('/question/add', methods=['POST'])
```

Comment:
The most import part is that the endpoint will need to work.

You can test your endpoint using `curl xxx` in your terminal. You can start with copying the curl in the api_list. And check the response from terminal 

Or try to learn how to use postman or similar tool to test